specutils/spectra/spectrum1d.py

```diff
@@ -91,17 +92,54 @@ def to_flux(self, unit):
             The converted flux array.
         """
         new_data = self.flux.to(
-            unit, equivalencies=eq.spectral_density(self.dispersion))
+            unit, equivalencies=eq.spectral_density(self.spectral_axis))
 
         self._data = new_data.value
         self._unit = new_data.unit
 
         return self.flux
 
-    def to_dispersion(self, unit, rest, type="doppler_relativistic"):
+    @property
+    def frequency(self):
```

Comment:
These should probably get docstrings that specify the units (so that the occasional user who actually reads the docs understands).  Bonus points for bits like "GHz are the default following a common radio convention" or similar